src/rules/__tests__/consistent-test-it.test.ts

```diff
@@ -11,90 +12,105 @@ ruleTester.run('consistent-test-it with fn=test', rule, {
   valid: [
     {
       code: 'test("foo")',
-      options: [{ fn: 'test' }],
+      options: [{ fn: TestCaseName.test }],
```

Comment:
Not a fan of these enums making everything so verbose (vs string union types) :|
But I guess that might just be a matter of preference

Reply:
We have to do it in the tests b/c they're in the rule.

I'm in the same boat as @SimenB - they're extra verbose, but I find the gain of being able to have them all tied back to a single thing that can be usage queried vs saving a few bytes worth it. 

You can have the best of both worlds by making the eum const.

Reply:
The TS lang server can actually show references (identical string literals)! :smile: 
But if there's a string literal `'test'` somewhere that is not used as a `TestCaseName` it'll still show up because it's still assignable to the type, so it might show too much.

But yeah as I said this is probably preference. Except for `const enum`, they're horrible because they cannot be compiled within the scope of a single source file ;D